README.md

```diff
@@ -137,6 +137,7 @@ Here are possible options:
 | `author_name`	| Author name for changelog. If not set, author of the merge commit is used	    | No |
 | `author_email`| Author email for changelog. If not set, author of the merge commit is used	| No |
 | `fedora`      | Whether to release on fedora. False by default | No |
+| `pypi`      | Whether to release on pypi. False by default | No |
```

Comment:
- I think we want it to be `True` by default. @TomasTomecek WDYT?
- can you move it above `fedora` option? The same in the `release-conf-example.yaml`

Reply:
- [ ] True by default. (waiting for further info)
- [x] Move it above fedora option.

Reply:
I agree with @jpopelka here; if we release the bot with pypi=False by default, we would need to update configs in all our repos (~10). Let's do pypi=True by default.

Reply:
On it.

Reply:
I was working on `configuration.py` and my changes weren't reflecting on `releasebot.py`. I found out the problem and is because of [this](https://github.com/user-cont/release-bot/blob/master/release_bot/releasebot.py#L27) import statement. 

My question is, why are we importing using the package name, and not just writing:

`import configuration` (they all are in the same folder).

Reply:
it is importing the configuration singleton: https://github.com/user-cont/release-bot/blob/master/release_bot/configuration.py#L137

Reply:
A very "basic" question. How do you verify and test the bot?

I had placed a `conf.yaml` in the `release-bot/release_bot/` directory and was running `releasebot.py` directly. This was working for me perfectly. 

Until today, I decided to update my system to Linux 5 (I was too eager), and now I am unable to do the same to test my changes. I get the following error message:

```
$ python releasebot.py -c conf.yaml --debug
Traceback (most recent call last):
  File "releasebot.py", line 27, in <module>
    from release_bot.cli import CLI
ModuleNotFoundError: No module named 'release_bot'
```

I also tried to run the tests, but they all failed with the same error message. I also tried the same thing on my Windows system, and it shows the same error message there. 
I also reverted to some earlier commits, to check if I haven't broken anything, but I have not (phew!).

So, can you tell me what I have been doing incorrectly, and why did it work for me previously?

Reply:
I figured out what I had done earlier. I had installed `release_bot` earlier, as the python package, and I had been importing the installed package's modules since then. After the update, I messed up my python packages installed in the user directory, and hence I was able to import the modules.
That is why I was not able to see the changes reflect when I was making changes in `configuration.py`

So, my question still remains. How do you test the code? Do you build it every time to test it?

Reply:
We run test suite inside of a container. See this make target for more information: https://github.com/user-cont/release-bot/blob/58c199dab3f134baf49357452c8d6d4fdb10889a/Makefile#L19

Otherwise if you want to test changes on your system and you don't want to install release-bot after every change you can take advantage of:
```
$ pip3 install -e .
# or 
$ python3 setup.py develop
```
Try to google those for more information.

Reply:
I test it directly from git usually:
```
$ PYTHONPATH=$PWD pytest-3 -k test_i_want_to_run
```

Reply:
> $ pip3 install -e .

Wow, I haven't known [that](https://stackoverflow.com/questions/19048732/python-setup-py-develop-vs-install), nice!